src/Sylius/Behat/Page/Admin/Order/ShowPage.php

```diff
@@ -392,6 +392,14 @@ public function hasBillingProvinceName($provinceName)
     /**
      * {@inheritdoc}
      */
+    public function hasIpAddressAssigner()
```

Comment:
Anyway, rewriting it to `getAssignedIP` will allow us for some flexibility, we should avoid creating page methods that return booleans.